server/src/test/java/org/apache/calcite/test/ServerTest.java

```diff
@@ -166,6 +166,42 @@ static Connection connect() throws SQLException {
     }
   }
 
+  @Test public void testCreateFunction() throws Exception {
+    try (Connection c = connect();
+         Statement s = c.createStatement()) {
+      boolean b = s.execute("create schema s");
+      assertThat(b, is(false));
+      try {
+        boolean f = s.execute("create function if not exists s.t \n" +
+                "as 'org.apache.calcite.udf.TableFun.demoUdf'\n" +
+                "using jar 'file:/path/udf/udf-0.0.1-SNAPSHOT.jar'");
+      } catch (SQLException e) {
+        assertThat(e.getMessage(),
+                containsString("Sql create function ddl is not support yet"));
+      }
+    }
+  }
+
+  @Test public void testDropFunction() throws Exception {
+    try (Connection c = connect();
+         Statement s = c.createStatement()) {
+      boolean b = s.execute("create schema s");
+      assertThat(b, is(false));
+
+      boolean f = s.execute("drop function if exists t");
+      assertThat(f, is(false));
+
+      try {
+        boolean f2 = s.execute("drop function t");
+        assertThat(f2, is(false));
+      } catch (SQLException e) {
+        assertThat(e.getMessage(),
+                containsString("Error while executing SQL \"drop function t\":" +
+                        " At line 1, column 15: Type 'T' not found"));
```

Comment:
We should also throw exception message as "XXX not support" instead "XXX not found".

Reply:
I added the default execution logic in SqlDrop for SqlDropFunction, so it will throw function not found in the root schema.

Reply:
Yean, i mean we do not even support "drop function" then why throws "xxx function not found" ?

Reply:
Oops, my mistake, you actually implement the "Drop function". Then we should add more tests about drop function:

**Positive tests**
1. Add a function in the schema and test remove with(without) if exists
2. Test drop function with specified catalog name and database name, i.e. `DROP FUNCTION catalog_name.db_name.func_name`

**Negative tests**
1. Test drop function what does not exist
2. Test drop function with case_sensive true but not found

Reply:
@danny0405 
Agree. If the create function is implemented, more test cases are needed. Currently, as the create function is not there, I can't add and drop function in test case. Thus, I would put the implementation of create and drop function in another PR, and leave this PR simple.

Reply:
We do not need the create function to be implemented, just put the function explicitly into the schema is ok, then we can add drop function test.

Reply:
@danny0405 I enhanced the test cases for drop function. But the namespace lookup of catalog_name.db_name is not handled for all table, function, and view. It requires a bigger change, so I would like to leave it as another PR. How do you think?

Reply:
I'm fine with the latest change, thanks.